reana_client/cli/workflow.py

```diff
@@ -271,19 +270,32 @@ def workflows_list(  # noqa: C901
             data.append(row)
 
         # Sort by given column, making sure that `None` is at the bottom of the list.
+        def get_sort_key(x, column_id):
+            if sort_columm_name == "run_number":
+                return list(map(int, x[column_id].split(".")))
+            elif sort_columm_name in workspace_size_header:
+                return x[column_id]["raw"]
+            return x[column_id] is not None, x[column_id]
+
         sort_column_id = 2
-        if sort_columm_name.lower() in headers[type]:
+        if sort_columm_name.lower() == "run_number":
```

Comment:
Very minor: I think this line is unnecessary? Since `headers[type].index("run_number")` is the same as `headers[type].index(sort_columm_name.lower())` or am I missing something here?